doc/classes/AudioEffectCapture.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectCapture" inherits="AudioEffect" version="4.0">
+	<brief_description>
+		Captures audio from an audio bus in real-time.
+	</brief_description>
+	<description>
+		AudioEffectCapture is an AudioEffect which copies all audio frames from the attached audio effect bus into its internal ring buffer.
+
+		Application code should consume these audio frames from this ring buffer using [code]get_buffer[/code] and process it as needed, for example to capture data from a microphone, implement application defined effects, or to transmit audio over the network.
```

Comment:
```suggestion
		[AudioEffectCapture] is an [AudioEffect] which copies all audio frames from the attached audio effect bus into its internal ring buffer.
		Application code should consume these audio frames from this ring buffer using [method get_buffer] and process it as needed, for example to capture data from a microphone, implement application defined effects, or to transmit audio over the network.
```
(note that I removed the empty line, each line == one paragraph in the in-editor and HTML output)